src/Middlewares/validation.js

```diff
@@ -0,0 +1,24 @@
+const jwt = require('jsonwebtoken')
+
+function validation(req, res, next) {
+    try {
+        const idReq = req.params.id
+        const {authorization: token} = req.headers;
+        const {id: idToken} = jwt.decode(token)
+        
+        if (idReq != idToken) throw new Error(
```

Comment:
@Deveckor Hay que utilizar la validación `!==`
```suggestion
        if (idReq !== idToken) throw new Error(
```